BibBuddy/app/src/main/java/de/bibbuddy/BookFragment.java

```diff
@@ -73,13 +99,18 @@ public void handleOnBackPressed() {
     shelfName = bundle.getString(LibraryKeys.SHELF_NAME);
     shelfId = bundle.getLong(LibraryKeys.SHELF_ID);
     bookModel = new BookModel(getContext(), shelfId);
+
     List<BookItem> bookList;
     bookList = bookModel.getBookList(shelfId);
+
     bookDao = bookModel.getBookDao();
     noteDao = bookModel.getNoteDao();
 
     exportBibTex = new ExportBibTex(StorageKeys.DOWNLOAD_FOLDER, shelfName);
 
+    bibTagValue = new HashMap<>();
+    importBibTex = new ImportBibTex(context, bibTags, bibTagValue);
```

Comment:
Why do you use `bibTags` and `bibTagValue` as parameters to create a new `ImportBibTex` Object? Isn't it better to use `bibTags` and `bibTagValue` directly in the `ImportBibTex` class?
```suggestion
    importBibTex = new ImportBibTex(context);
```